go-controller/pkg/metrics/ovn.go

```diff
@@ -86,6 +89,30 @@ var metricBridgeMappings = prometheus.NewGaugeVec(prometheus.GaugeOpts{
 	},
 )
 
+var metricServiceSubnet = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+	Namespace: MetricOvnNamespace,
+	Subsystem: MetricOvnSubsystemController,
+	Name:      "service_subnet",
+	Help: "A metric with a constant '1' value labeled by subnet that " +
+		"specifies the subnet used for service.",
+},
+	[]string{
+		"subnet",
```

Comment:
Does it make more sense to split the labels into `subnet_ipv4` and `subnet_ipv6`? If someone decides to perform prometheus queries against these metrics, I think they will find it easier to work with if it's split rather than a merged string.

Reply:
+1.

Reply:
Done